src/map/utils/mobutils.cpp

```diff
@@ -231,68 +203,103 @@ namespace mobutils
         JOBTYPE   mJob     = PMob->GetMJob();
         JOBTYPE   sJob     = PMob->GetSJob();
         uint8     mLvl     = PMob->GetMLevel();
+        uint8     sLvl     = PMob->GetSLevel();
         ZONE_TYPE zoneType = PMob->loc.zone->GetType();
 
+        uint8 grade;
+        uint8 gradesj;
```

Comment:
My last nitpick: We can use verbose varibale names.

I mean, i can understand its the grade used for subjob, in this case, but for the sake of everyone else that will look/use this file after us, the moredescriptive a variable name is, the easier it is to understand.